README.md

```diff
@@ -2,6 +2,17 @@
 
 A CLI tool to simplify your release process!
 
+**Important note**:
+this tool simplify a process, if you don't have one, it is the right time to adopt one!
+
+## The Release Process
+
++ use semver
++ never commit to master, except bump of version
```

Comment:
This happens from time to time. Instead of referencing the PR number, we should be referencing the commit sha.

Reply:
I don't get the point here: would you like to add to the output release string, the commits sha?